src/utilities/indexer/updateAttestationCount.ts

```diff
@@ -0,0 +1,52 @@
+import { type ICType } from '@kiltprotocol/sdk-js';
+
+import { Op } from 'sequelize';
+
+import { CType as CTypeModel } from '../../models/ctype';
+
+import { logger } from '../logger';
+
+import { matchesGenerator } from './queryFromIndexer';
+
+export async function updateAttestationsCount() {
+  const fields = ['cTypeId: id', 'attestationsCreated', 'registrationBlockId'];
+
+  const queryParams = {
+    entity: 'cTypes',
+    alias: 'attestationsCount',
+    fields,
+  };
+
+  interface QueriedAttestationCount {
+    cTypeId: ICType['$id'];
+    attestationsCreated: number;
+    registrationBlockId: string; // Block Ordinal Number, without punctuation
+  }
+  const entitiesGenerator =
+    matchesGenerator<QueriedAttestationCount>(queryParams);
+
+  for await (const entity of entitiesGenerator) {
+    const { cTypeId, attestationsCreated } = entity;
+
+    const cTypeToUpdate = await CTypeModel.findOne({
+      where: {
+        id: {
+          [Op.eq]: cTypeId,
+        },
+        attestationsCount: {
+          [Op.ne]: attestationsCreated,
+        },
+      },
+    });
+
+    if (!cTypeToUpdate) {
+      continue;
+    }
```

Comment:
If you want to reduce the number of database operations this way, I can recommend a [more efficient approach](https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#simple-update-queries)

Reply:
Ok, used `update` instead. 

The log message got a bit worse but not a biggie.

Reply:
I would like to revert this change. I'm really interested on those log messages at the time of merge. 

Not only to track changes here, but also as a test for the indexer.

Reply:
I am okay with that